content/docs/2.8/scalers/azure-pipelines.md

```diff
@@ -97,6 +146,32 @@ spec:
     metadata:
       poolID: "1"
       organizationURLFromEnv: "AZP_URL"
+      parent: "example-keda-template"
+      demands: "maven,docker"
     authenticationRef:
      name: pipeline-trigger-auth
 ```
+###Example for Parent Deployment or StatefulSet
```

Comment:
I'm not sure about having the requirement of an agent for doing nothing TBH. I prefer to register the agent manually in the AzDO UI than creating a deployment only for that.
WDYT @kedacore/keda-contributors ?

Reply:
My concern is that you have your own business logic to manage this and it works with your use case, but it's managed by your side, not by KEDA. IMO,  we should be most agnostics as possible and use the upstream tools for doing the upstream things. Obviously, we can add a "tip" suggesting the approach, having to deploy (and potentially delete) the deployment only to upgrade the agent in AzDO could be a complex action if you only have 2-3 different agents.

Maybe I'm overthinking about this and that's why I'd like to get the opinion of other @kedacore/keda-contributors but it's more a thing about how we document the feature than an issue with the feature itself (In fact, it's already approved :) ).